packaging/rpm/make-rpm.sh

```diff
@@ -68,6 +69,13 @@ EOF
   rpmbuild --quiet ${RPMBUILD_OPT} --define "_topdir ${RPMBUILD_DIR}" --define "_binary_payload w19T8.zstdio" "${RPMBUILD_DIR}"SPECS/microshift.spec
 }
 
+print_info() {
+  title "RPM info"
+  for dir in ${RPM_INFO_DIRS}; do
+    find "${RPMBUILD_DIR}${dir}" -type f -exec sh -c 'i=$1; echo "${i}" && rpm -qip --dump "${i}" && echo' shell {} \;
```

Comment:
Can you provide a log of how this looks like in practice?

Reply:
The output will look something like https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/44677/rehearse-44677-pull-ci-openshift-microshift-release-4.14-test-rpm/1716393559587819520/artifacts/test/build-log.txt

The idea is to verify that `make rpm` actually builds something.

Can we do something to report an error if no RPM files are found? Ideally `make rpm` would have already failed, but just in case.

Reply:
Keep in mind that this is part of the same script that builds the RPMs, its just a post step running only if an env var is defined.
Do we foresee any situation in which the [rpmbuild](https://github.com/openshift/microshift/blob/main/packaging/rpm/make-rpm.sh#L68) will build nothing and not fail?

As for the output, it looks just like what Doug linked but with an additional title.